content/software/ide-v2/tutorials/getting-started/02.ide-v2-uploading-a-sketch/ide-v2-uploading-a-sketch.md

```diff
@@ -36,10 +36,14 @@ Uploading a sketch is quick and easy, but let's take a look at what we need to d
 
 ![Verifying and uploading buttons.](assets/uploading-a-sketch-img01.png)
 
-**3.** First, we need to select the board that we are going to use. We can do this by navigating to **Tools > Port > {Board}**. The board(s) that are connected to your computer should appear here, and we need to select it by clicking it. In this case, our board is displayed as **COM17 (Arduino MKR WiFi1010)**.
+**3.** But before doing any of that, we should choose what board we are uploading to. Beside the verify and upload button you should see a a drop down menu, this will display any Arduino board that is connected to your computer. 
 
 ![Selecting the board.](assets/uploading-a-sketch-img03.png)
 
+So connect your board to the computer and select it from the drop down menu. You'll know that there is a connection to the board when the board name appears in **bold** and there's a USB icon next to it.

```

Comment:
The icon is dependent on the port's protocol. The USB icon is used only for ports of the `serial` protocol. Although that is the most common protocol, there are other protocols. There is a dedicated icon for `network` protocol ports and then a catch-all cube icon for ports of any other protocol (e.g., `teensy`).

The infrastructure allows adding icons for other common protocols (e.g., [`bluetooth` is already in place](https://github.com/arduino/arduino-ide/blob/2.1.0/arduino-ide-extension/src/browser/boards/boards-toolbar-item.tsx#L307-L312))